src/components/form/form.js

```diff
@@ -237,6 +248,7 @@ class Form extends React.Component {
   static defaultProps = {
     buttonAlign: 'right',
     cancel: true,
+    savePrompt: true,
```

Comment:
Im not sure this is the best name as its not prompting you when you save. Trying to think of something better now

`dirtyFormWarning`
`dirtyFormCheck`
`warnOnDirtyForm`

Reply:
`unsavedWarning`?